Sources/vincenty/vincenty.swift

```diff
@@ -15,6 +20,11 @@ public let wgs84 = (a: 6378137.0, f: 1 / 298.257223563)
 /// π (for convenience)
 private let pi = Double.pi
 
+public struct VincentyInverseResults {
+    var distance:Double = 0.0
+    var azimuths:(initialTrueTrack:Double, finalTrueTrack:Double)
```

Comment:
Thanks for implementing all the PR remarks. One last question on the naming here. 
I tried to find a reference for the `...trueTrack` naming, but did not find any domain unspecific. 
The source you added above http://www.movable-type.co.uk/scripts/latlong-vincenty.html calls the azimuths initial and final bearing. https://en.wikipedia.org/wiki/Vincenty%27s_formulae calls them just azimuths. 
I think one could safely stick to an unnamed tuple `azimuths:(Double, Double)`. The term azimuths can be found in both sources. That the order corresponds to the order of input points should be self-explanatory. 
You could still unpack to 
```
let (initialTrueTrack, finalTrueTrack) = azimuths
```
Others may simply use the short syntax, `azimuths.0` and `azimuths.1`.

Did you consider the tuple approach I suggested earlier for the entire result? From my perspective, it has the upside that you can do tuple unpacking like 
```
let (distance, azimuths) = try! vincenty.solveInverse(x, y)
```
or even 
```
let (distance, azimuths: (initialTrueTrack, finalTrueTrack)) = try! vincenty.solveInverse(x, y)
```
The `VincentyInverseResults` struct seems to be a simple record (or DTO) by design, so I don't see any benefit over a tuple unless some logic should be implemented in that class.

Reply:
Roger, this has been fixed now.